ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -7387,3 +7514,23 @@ NO2C2XDSP:
   - zone_air_no2_concentration_sensor_2
   implements:
   - CONTROL
+
+HCPC2X:
+  guid: "67a960f5-65e7-400f-b64e-ab4ef1ae9035"
+  description: "Circulation control for 2 heating water pumps part of a hydronic plant."
```

Comment:
Are these on-board pumps? If so, they aren't really part of the plant, they are part of the equipment